### PR TITLE
Ported to Python

### PR DESCRIPTION
**Pull request by DongDuong9E**
Why did I make this

### CODE REVIEW

**Comment by sr229** [comment]
@martmists get your ass in here

File: src/Lolicon.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import subprocess
+# Lolicon error handling
+# Wank wank dicks dicks
+
+class LoliconException(Exception):
+   """Raised when there is a fucking lolicon"""
```
Comment (martmists) [inline src/Lolicon.py:6]: Use 4-space indents

File: src/Lolicon.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import subprocess
```
Comment (martmists) [inline src/Lolicon.py:1]: Lowercase file name pls

File: src/Lolicon.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# Lolicon error handling
+# Wank wank dicks dicks
+
+class LoliconException(Exception):
+   """Raised when there is a fucking lolicon"""
+   pass
+
+# Lolicon Main
```
Comment (martmists) [inline src/Lolicon.py:9]: missing a main() method which needs to be called from an `if __name__ == "__main__":` block

File: src/Lolicon.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+# Me me big disappointment
+
+def intice():
+   """do nothing because weebs are already attracted to underage anime girls"""
+   pass
+
+def callthefbi(age):
```
Comment (martmists) [inline src/Lolicon.py:16]: Type hints for age

File: src/Lolicon.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+   if age < 15:
+       subprocess.run(["call_911"], capture_output=True)
```
Comment (martmists) [inline src/Lolicon.py:18]: subprocess.run is considered unsafe

Comment (sr229) [reply src/Lolicon.py:18]: Pretty sure because this was the only API equivalent of `NodeJSProcess.child_process()`.

File: src/Lolicon.py (hunk 2 above)
Comment (DongDuong9E) [reply src/Lolicon.py]: Renamed Lolicon.py to lolicon.py

File: src/Lolicon.py (hunk 4 above)
Comment (DongDuong9E) [reply src/Lolicon.py]: Added type hints for age in the new commit.

File: src/Lolicon.py (hunk 3 above)
Comment (DongDuong9E) [reply src/Lolicon.py]: Added this in the new pull request

File: src/lolicon.py [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    pass
+
+def callthefbi(age: int):
+    if age < 15:
+        subprocess.run(["call_911"], capture_output=True)
+    else:
+        raise LoliconException("Probably just something weird happened.")
+
+def main():
+    age = input("What's the girl's age? ")
```
Comment (martmists) [inline src/lolicon.py:23]: Age is a string here and the comparison on line 15 will fail

File: src/lolicon.py [hunk 7]
```diff
@@ -0,0 +1,27 @@
+import subprocess
```
Comment (martmists) [inline src/lolicon.py:1]: Missing shebang line

File: src/lolicon.py [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+    pass
+
+# Lolicon Main
+# Me me big disappointment
+
+def intice():
+    """do nothing because weebs are already attracted to underage anime girls"""
+    pass
+
+def callthefbi(age: int):
```
Comment (martmists) [inline src/lolicon.py:16]: underscores in function name

File: src/Lolicon.py (hunk 5 above)
Comment (martmists) [reply src/Lolicon.py]: subprocess.Popen is a better solution as it gives more control.

File: src/lolicon.py [hunk 9]
```diff
@@ -13,15 +14,15 @@ def intice():
     """do nothing because weebs are already attracted to underage anime girls"""
     pass
 
-def callthefbi(age: int):
+def _callthefbi(age: int):
```
Comment (martmists) [inline src/lolicon.py:17]: what the fuck

Comment (DongDuong9E) [reply src/lolicon.py:17]: please elaborate

Comment (martmists) [reply src/lolicon.py:17]: why add an underscore before the name if it has no need to be "private"

Comment (DongDuong9E) [reply src/lolicon.py:17]: https://github.com/ClarityMoe/lolicon/pull/8#discussion_r419449748 You asked for underscore ¯\_(ツ)_/¯

Comment (martmists) [reply src/lolicon.py:17]: I meant call_the_fbi to match snake_case, as defined in [PEP 0008](https://www.python.org/dev/peps/pep-0008/)

Comment (DongDuong9E) [reply src/lolicon.py:17]: I see

**Comment by sr229** [comment]
Bruh

**Pull request merged**